src/components/container/IntradayHeartRateChart/IntradayHeartRateChart.data.ts

```diff
@@ -0,0 +1,44 @@
+import { add, addMinutes, startOfDay } from "date-fns";
+import { DeviceDataV2Aggregate } from "@careevolution/mydatahelps-js";
+export type IntradayHeartRateData = { [key: string]: DeviceDataV2Aggregate };
+const today = startOfDay(new Date());
+
+function getRandomInt(min : number, max : number) {
+    min = Math.ceil(min);
+    max = Math.floor(max);
+    return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+
+function loadForTest(data: DeviceDataV2Aggregate[]) {
+    var testData : IntradayHeartRateData = {};
+    data.forEach((data) => {
+        testData[(new Date(data.date)).getTime()] = data;
+    });
+    return testData;
+}
+
+let testData: DeviceDataV2Aggregate[] = [];
+for (let i = 0; i < 48; i++) {
+    testData.push({ participantID: "ABCDEFG", participantIdentifier: "ABCDEFG", date: addMinutes(today, (i * 5)).toISOString(), statistics: { "avg": getRandomInt(80, 185) } });
+}
+const halfDayData = loadForTest(testData);
```

Comment:
Nit:  Can we generate data that more closely matches a typical day of heart rate data?  Maybe an eight hour period with consistent low numbers, an eight hour period with normal numbers, and an eight hour period with higher activity and a few spikes?